larq_zoo/core/utils.py

```diff
@@ -170,3 +168,29 @@ def decode_predictions(preds, top=5, **kwargs):
     ValueError: In case of invalid shape of the `pred` array (must be 2D).
     """
     return keras_decode_predictions(preds, top=top, **kwargs)
+
+
+def TFOpLayer(tf_op):
```

Comment:
```suggestion
def TFOpLayer(tf_op) -> tf.keras.layers.Layer:
```

Reply:
Ah except this return type is wrong, should be `-> Callable[..., tf.keras.layers.Lambda]` I think?